extensions/MozChangeField/template/en/default/hook/bug_modal/header-end.html.tmpl

```diff
@@ -0,0 +1,9 @@
+[%# This Source Code Form is subject to the terms of the Mozilla Public
+  # License, v. 2.0. If a copy of the MPL was not distributed with this
+  # file, You can obtain one at http://mozilla.org/MPL/2.0/.
+  #
+  # This Source Code Form is "Incompatible With Secondary Licenses", as
+  # defined by the Mozilla Public License, v. 2.0.
+  #%]
+
+[% javascript_urls.push("extensions/MozChangeField/web/js/severitys1priorityp1.js") %]
```

Comment:
That filename looks like soup, `severity-s1-priority-p1.js` would be much easier to read.